wheel/src/api.rs

```diff
@@ -385,6 +437,11 @@ pub fn chia_rs(_py: Python<'_>, m: &Bound<'_, PyModule>) -> PyResult<()> {
     m.add_function(wrap_pyfunction!(confirm_included_already_hashed, m)?)?;
     m.add_function(wrap_pyfunction!(confirm_not_included_already_hashed, m)?)?;
 
+    // multithread validattion
```

Comment:
I think this comment is misleading. These functions are just plain functions, there's no threading involved here